src/DynamoCore/Graph/Workspaces/HomeWorkspaceModel.cs

```diff
@@ -18,6 +18,9 @@
 
 namespace Dynamo.Graph.Workspaces
 {
+    /// <summary>
+    ///     This class represents executable workspace.
```

Comment:
This class contains methods and properties that defines a (use cref) workspacemodel.